.p4a

```diff
@@ -7,7 +7,7 @@
 --orientation sensor
 --requirements python3,android,pyjnius,genericndkbuild,sqlite3,cryptography,twisted,attrs,bcrypt,service_identity,pyasn1,pyasn1_modules,pyopenssl,openssl,six,liblzma,zstandard,beautifulsoup4==4.10.0,zimply_core,kolibri_explore_plugin~=3.0,kolibri_zim_plugin
 --android-api 30
---minsdk 21
+--minsdk 24
```

Comment:
I think you're going to have to add `--allow-minsdk-ndkapi-mismatch` to this file to avoid some bogus advice from p4a. If you look at the NDK 23 [changelog](https://github.com/android/ndk/wiki/Changelog-r23), you can see that it supports all the way back to API 16. I don't know why p4a suggests 21 as the minimum NDK API.

Reply:
Alternatively, you could add `--ndk-api 24`, I believe. I actually don't get the point of the "NDK API" value at all. The only way it affects the build is the version checking guards that p4a does.